web/items.go

```diff
@@ -0,0 +1,88 @@
+package web
+
+import (
+	"log"
+	"net/http"
+
+	"time"
+
+	"github.com/btshopng/btshopng/config"
+	"github.com/btshopng/btshopng/models"
+	uuid "github.com/satori/go.uuid"
+)
+
+func NewItemHandler(w http.ResponseWriter, r *http.Request) {
+
+	data, err := GetProfileData(r)
```

Comment:
Are you suggesting i use:
`
user := models.User{}
user.Get(config.Get(),'userid)
`
instead of 
`user, err := Userget(r)`